src/arkouda_server.chpl

```diff
@@ -124,6 +123,42 @@ proc main() {
         stdout.flush();
     }
 
+    /**
+     * Register our server commands in the CommandMap
+     * There are 3 general types
+     * 1. Standard, required commands which adhere to the standard Message signature
+     * 2. Specialized, required commands which do not adhere to the standard Message signature
+     * 3. "Optional" modules which are included at compilation time via ServerModules.cfg
+     */
+    proc registerServerCommands() {
+        registerBinaryFunction("tondarray", tondarrayMsg);
+        registerFunction("create", createMsg);
+        registerFunction("delete", deleteMsg);
+        registerFunction("arange", arangeMsg);
+        registerFunction("linspace", linspaceMsg);
+        registerFunction("set", setMsg);
+        registerFunction("info", infoMsg);
+        registerFunction("str", strMsg);
+        registerFunction("repr", reprMsg);
+        registerFunction("getconfig", getconfigMsg);
+        registerFunction("getmemused", getmemusedMsg);
+        registerFunction("getCmdMap", getCommandMapMsg);
+        registerFunction("clear", clearMsg);
+
+        // For a few specialized cmds we're going to add dummy functions, so they
+        // get added to the client listing of available commands. They will be
+        // intercepted in the cmd processing select statement and processed specially
+        registerFunction("array", akMsgSign);
+        registerFunction("connect", akMsgSign);
+        registerFunction("disconnect", akMsgSign);
+        registerFunction("noop", akMsgSign);
+        registerFunction("ruok", akMsgSign);
+        registerFunction("shutdown", akMsgSign);
```

Comment:
`shutdown` is the only specialized command not listed in the `select cmd` loop below, so I just wanna verify that's it's being handled. I'm pretty sure it is because it doesn't look like it was present before, but i figure it doesn't hurt to check

Reply:
Yup, it's handled early see [line 319](https://github.com/Bears-R-Us/arkouda/blob/6879dcbf26f0686854b4b6040e43acd33c8a1020/src/arkouda_server.chpl#L319)

Good to double check :+1: